kong/plugins/acl/api.lua

```diff
@@ -1,9 +1,11 @@
 local endpoints = require "kong.api.endpoints"
-local responses = require "kong.tools.responses"
 
+
+local kong             = kong
```

Comment:
Is it okay to `local`-ize the `kong` global? I am not sure if it is, because in the future different versions of the PDK may be active, and the appropriate PDK version might be determined only after the module is `require`d... or are we avoiding this issue via metatable indirection? do we have a policy on that? pinging @thibaultcha

Reply:
It should be fine yes :+1: The reference to the `kong` global should always be the same, and underlying metatables should be responsible for redirecting callers to the appropriate PDK version (or cached modules that are rotated, like the current logging facilities implementation). The "don't" would be to cache individual fields of the `kong` global (e.g. `kong.pdk_version`, or `kong.request.get_path`).